modules/manage/pages/kubernetes/kubernetes-rack-awareness.adoc

```diff
@@ -37,7 +41,7 @@ example-worker2      us-east-1b
 example-worker3      us-east-1c
 ----
 
-If you don't see any values in the Zone column, make sure to annotate your Node resources with key/value pairs that correspond to your fault-tolerance requirements. For example:
+If you don't see any values in the Zone column, make sure to annotate or label your Node resources with key/value pairs that correspond to your fault-tolerance requirements. For example:
```

Comment:
I'm not sure if we should help customer with automation that leverages DaemonSet that annotates Kubernetes Node resource with correct ZONE using AWS metadata. 

As an example:

```yaml
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: labeler
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: labeler
rules:
  - apiGroups:
      - ""
    resources:
      - nodes
    verbs:
      - patch
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: labeler
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: labeler
subjects:
  - kind: ServiceAccount
    name: labeler
    namespace: $PLACEHOLDER_TO_CHANGE
---
apiVersion: apps/v1
kind: DaemonSet
metadata:
  name: labeler
spec:
  selector:
    matchLabels:
      name: labeler
  template:
    metadata:
      labels:
        name: labeler
    spec:
      serviceAccountName: labeler
      initContainers:
        - name: labeler
          image: debian:bullseye-slim
          imagePullPolicy: IfNotPresent
          command:
            - /bin/bash
            - -c
            - --
          args:
            - |
              apt-get update -y && apt-get install -y curl jq apt-transport-https ca-certificates
              curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
              # This overwrites any existing configuration in /etc/apt/sources.list.d/kubernetes.list
echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/v1.28/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list
              apt-get update -y && apt-get install -y kubectl
               
              # Get a token to be able to interact with the EC2 instance metadata API v2
              # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html
              TOKEN=$(curl -X PUT "http://169.254.169.254/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 21600")
              # Get the current node's AZ ID
              AZ_ID=$(curl -H "X-aws-ec2-metadata-token: $TOKEN" -v "http://169.254.169.254/latest/meta-data/placement/availability-zone-id")
              kubectl label node/"$HOST" "topology.cloud.redpanda.com/zone-id=$AZ_ID" --overwrite
      containers:
        - name: pause
          image: debian:bullseye-slim
          imagePullPolicy: IfNotPresent
          command:
            - /bin/bash
            - -c
            - --
          args:
            - |
              trap : TERM INT; sleep infinity & wait
```

I need to admit I didn't test the above.

Reply:
This would be specific to AWS though. What about users in other cloud environments? Also, we say that:

> Managed Kubernetes platforms usually annotate Node resources with the availability zone in which the node instance is hosted. For example `topology.kubernetes.io/zone=use-az1`.

Reply:
Kudos to Juan Castillo

> This change is not required in GCP since GKE already labels node with
the right value for the topology.kubernetes.io/zone label, which in
the case of GCP is the fully-qualified zone name.